src/community/roadmap/index.md.njk

```diff
@@ -28,24 +28,25 @@ We recently:
 - released [GOV.UK Frontend v4.0.1](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.1)
 - updated the [Accessible autocomplete](https://github.com/alphagov/accessible-autocomplete/releases/tag/v2.0.4) component and clarified [future plan](https://github.com/alphagov/accessible-autocomplete/issues/532)
 - made changes to [Start page](https://github.com/alphagov/design-system-team-internal/issues/552) pattern
```

Comment:
```suggestion
- made changes to the [Start page](https://github.com/alphagov/design-system-team-internal/issues/552) pattern
```
Also, is this the Start page pattern or the Start using a service pattern?

Reply:
Looks like Start using a service according to this PR: https://github.com/alphagov/govuk-design-system/pull/2086/files